app/pulp/app/models/progress.py

```diff
@@ -54,6 +56,7 @@ class ProgressReport(Model):
     done = models.IntegerField(default=0)
 
     task = models.ForeignKey("Task", on_delete=models.CASCADE)
+    suffix = models.TextField(default=None)
```

Comment:
@ipanova oh that is right. In this case I think having it default to empty string would be the best. Since string are immutable a default of `default=''` would work fine.